Resources/doc/reference/security.rst

```diff
@@ -157,6 +158,9 @@ So our ``security.yml`` file may look something like this:
                 ROLE_ADMIN:             [ROLE_STAFF, ROLE_SONATA_FOO_EDITOR, ROLE_SONATA_FOO_ADMIN]
                 ROLE_SUPER_ADMIN:       [ROLE_ADMIN, ROLE_ALLOWED_TO_SWITCH]
 
+                # you could alternatively use for an admin who has all rights 
+                ROLE_ALL_ADMIN:         [ROLE_STAFF, ROLE_SONATA_FOO_ALL ]
```

Comment:
* `ROLE_ALL_ADMIN` is a confused example IMO. When I read this, I understand that I can go to every admin part and not only the `FOO` part. That's sounds like `ROLE_SUPER_ADMIN`.
* `ROLE_SONATA_FOO_ALL ]` -> `ROLE_SONATA_FOO_ALL]`